analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -190,15 +192,35 @@ class SpdxDocumentFile(
             Package(
                 id = pkg.toIdentifier(),
                 declaredLicenses = sortedSetOf(pkg.licenseDeclared),
-                concludedLicense = pkg.licenseConcluded.toSpdx(),
+                concludedLicense = getConcludedLicense(pkg),
                 description = packageDescription,
                 homepageUrl = pkg.homepage.mapNotPresentToEmpty(),
                 binaryArtifact = RemoteArtifact.EMPTY, // TODO: Use "downloadLocation" or "externalRefs"?
-                sourceArtifact = RemoteArtifact.EMPTY, // TODO: Use "sourceInfo" or "externalRefs"?
+                sourceArtifact = buildSourceArtifact(pkg),
                 vcs = VersionControlSystem.forDirectory(packageDir)?.getInfo() ?: VcsInfo.EMPTY
             )
         }
 
         return listOf(ProjectAnalyzerResult(project, packages))
     }
+
+    /**
+     * Return the concluded license to be used in ORT's data model, which expects a not present value to be null
+     * instead of NONE or NOASSERTION.
+     */
+    private fun getConcludedLicense(pkg: SpdxPackage): SpdxExpression? =
```

Comment:
@sschuberth @tsteenbe @mnonnenmacher : Shall we file a ticket for making "NONE" a valid value for the package concluded license in ORT?

Note that for license finding curations we already do allow that.

Reply:
I'm a bit undecided. On the one hand it would be consistent to align on SPDX expressions for "everything about licenses". On the other hand, checking for (not) `null` is much more convenient than checking for `isPresent()` / `isNotPresent()`. My hunch is that we should still align on SPDX expression, but I'd like to also hear @mnonnenmacher opinion on that once he's back.

Reply:
I've filed #3122 so we can have the discussion there later on.